src/pages/Donate/Donate.jsx

```diff
@@ -14,6 +14,40 @@ export default function Donate() {
 		updateDonateAmount(event.target.value);
 	};
 
+	useEffect(() => {
+        const paypalScript = document.createElement("script");
+        paypalScript.src = "https://www.paypal.com/sdk/js?client-id=ATMg6l2UNbTrMPc99AbNMe_eT8PJ2QfJ1lRGzeZPh4wl8BampZl2YutnUid-3xgU57FOZFq1rjZwUIru&currency=USD"; // Replace YOUR_CLIENT_ID
+        paypalScript.async = true;
+        paypalScript.onload = () => {
+            if (window.paypal) {
+                window.paypal.Buttons({
+                    createOrder: (data, actions) => {
+						const sanitizedAmount = donateAmount.replace("$", "");
+						if (!sanitizedAmount || isNaN(sanitizedAmount)) {
+							alert("Please select a valid donation amount.");
+							return;
+						}
+						return actions.order.create({
+							purchase_units: [
+								{
+									amount: {
+										value: sanitizedAmount || "10", // Default to $10 if no amount is selected
```

Comment:
In what situation would we need the default value? If we return early on condition !sanitizedAmount then would we ever reach the default value?

Reply:
This makes sense. Default value is not needed when the input string is wrong.